h2o-py/h2o/tree/tree.py

```diff
@@ -29,37 +29,113 @@ def __init__(self, model, tree_number, tree_class=None):
 
     @property
```

Comment:
Also at the top, can you add parameters under the H2OTree section (before "left_children")? 
Currently, the function is specified as "class h2o.tree.H2OTree(model, tree_number, tree_class=None)" but model, tree_number, and tree_class are not described.

From the R documentation:
model - The model this tree is related to.
tree_number - An integer representing the order in which the tree has been built in the model.
tree_class - A string representing the name of the tree's class. The number of tree classes equals the number of levels in categorical response column. As there is exactly one class per categorical level, the name of the tree's class is equal to the corresponding categorical level of the response column. In case of regression and binomial models, the name of the categorical level is ignored and can be omitted, as there is exactly one tree built in both cases.